packages/manager/modules/cloud-universe-components/src/helper/order/service.js

```diff
@@ -161,6 +161,9 @@ export default class CucOrderHelperService {
           case 'QC':
             targetURL = 'https://ca.ovh.com/fr';
             break;
+          case 'US':
+            targetURL = 'https://ovh.us';
```

Comment:
Should not be better to use `https://us.ovhcloud.com/` instead ?

Reply:
Confirmed with the PO that this link is the right one